### PR TITLE
(feature)(hidden)translations

### PR DESCRIPTION
**Pull request by dmytroshch**
ticket - https://app.asana.com/0/0/1200581516527580/f

Снимок экрана от 2021-08-31 19-26-10

1. Connected i18next plugins 
2. Added config
3. Added en keys for Auth page
4. Connected with ufx-ui components (https://github.com/bitfinexcom/ufx-ui/pull/115)
5. Connected with bfx-hf-server and bfx-hf-algo (https://github.com/bitfinexcom/bfx-hf-server/pull/118 https://github.com/bitfinexcom/bfx-hf-algo/pull/184)

### CODE REVIEW

**Comment by bhoomij** [comment]
@dmytroshch please mark as DRAFT

**Comment by JasonMHasperhoven** [comment]
So far looking good :+1:

File: src/redux/middleware/ws/on_message.js [hunk 1]
```diff
@@ -186,73 +195,74 @@ export default (alias, store) => (e = {}) => {
...
-        const [,, aos] = payload
+        const [, , aos] = payload
         store.dispatch(WSActions.recvDataAlgoOrders({ aos }))
         break
       }
 
       case 'data.ao': {
-        const [,, ao] = payload
+        const [, , ao] = payload
+        console.log(ao, 'ao')
```
Comment (Blaumaus) [inline src/redux/middleware/ws/on_message.js:259]: probably should be removed:
```suggestion
```

File: src/components/TimeFrameDropdown/TimeFrameDropdown.js [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-      value: timeFrames[tfLabels[i]],
-    })
-  }
+  const { t } = useTranslation()
+  const options = useMemo(() => {
+    return _map(timeFrames, (time) => ({
+      value: time,
+      label: t(`time.${time}`),
+    }))
+  }, [])
```
Comment (Blaumaus) [inline src/components/TimeFrameDropdown/TimeFrameDropdown.js:16]: shouldn't we add `t` here as a dependency, so the options will be updated when the language changes?
```suggestion
  }, [t])
```

File: src/components/CcyInfoModal/CcyInfoModal.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ const CcyInfoModal = ({
   }, [isModalVisible])
 
   useEffect(() => {
-    if (!body) {
+    if (!body || !containerRef || !containerRef.current) {
```
Comment (Blaumaus) [inline src/components/CcyInfoModal/CcyInfoModal.js:27]: why do we need `|| !containerRef || !containerRef.current` here?
```suggestion
    if (!body) {
```

**Review by Blaumaus — CHANGES_REQUESTED** [review]
looks great to me with some minor proposals

**Review by Blaumaus — CHANGES_REQUESTED** [review]
I also noticed this strange glitch when the Russian and English translations at Ticker Panel got desynchronised 

https://user-images.githubusercontent.com/35810911/134165247-c2d40454-3ba6-4a6c-b5f9-111cb25759e8.mp4

File: src/components/CcyInfoModal/CcyInfoModal.js (hunk 3 above)
Comment (dmytroshch) [reply src/components/CcyInfoModal/CcyInfoModal.js:27]: sometimes app crashes without this condition

**Comment by dmytroshch** [comment]
> I also noticed this strange glitch when the Russian and English translations at Ticker Panel got desynchronised
> bfx-hf-ui-core-translations-glitch-2021-09-21_14.26.28.mp4

resolved https://github.com/bitfinexcom/ufx-ui/pull/119

**Pull request merged**